terra/core/settings.py

```diff
@@ -264,7 +264,7 @@ def need_to_set_virtualenv_dir(self):
     {
       "logging": {
         "level": "ERROR",
-        "format": f"%(asctime)s (%(hostname)s:%(zone)s): %(levelname)s - %(filename)s - %(message)s",
+        "format": "%(asctime)s (%(hostname)s:%(zone)s): \x1b[31m%(levelname)s/%(processName)s\x1b[0m - %(filename)s - %(message)s",
```

Comment:
what happens if the tty doesn't support color

Reply:
1) You can _always_ support color, `less -R` :) /s
1) Terminals support color, not tty. You are thinking of the istty check for color, which people commonly do so when you are pipeing, you don't get color codes in there, making `sed` and `grep` difficult. The problem with that check is it becomes impossible to pipe colors if you actually _wanted_ to, so I believe there should always be a force option.
2) that was for my debugging purposes only, and should be removed

It we want to support color, there is a way to add a "Formatter" to the logger to do that, I think. But this is not currently in the high priority wish list. I would love to have this, of course. This little POC was extrememely useful in skimming the logs.

Reply:
ya. that would be nice